web/app/scripts/views/record/add-edit-partial.html

```diff
@@ -5,6 +5,83 @@
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-12">
+                            <div class="form-group">
+                                <label class="control-label">Label</label>
+                                <input type="text" class="form-control" ng-model="ctl.record.label">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-6">
+                            <div class="form-group">
+                                <label class="control-label">Latitude</label>
+                                <input type="number" class="form-control"
+                                    ng-change="ctl.onGeomChanged()" ng-model="ctl.geom.lat">
+                            </div>
+                        </div>
+                        <div class="col-md-6 pull-right">
+                            <div class="form-group">
+                                <label class="control-label">Longitude</label>
+                                <input type="number" class="form-control"
+                                    ng-change="ctl.onGeomChanged()" ng-model="ctl.geom.lng">
+                            </div>
+                        </div>
+                        <div class="col-md-12">
+                            <div class="map" leaflet-map driver-embed-map style="height:300px;"></div>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-6">
+                            <div class="form-group">
+                                <label class="control-label">occurred From</label>
+                                <div class="input-group" id="datetimepicker-From">
+                                    <input type="text" class="form-control"
+                                        datetimepicker
+                                        datetimepicker-options="{{ ctl.occurredFromOptions }}"
+                                        ng-change="ctl.occurredFromChanged()"
+                                        ng-model="ctl.record.occurred_from"
+                                        placeholder="From">
+                                    <span class="input-group-addon">
+                                        <span class="glyphicon glyphicon-calendar"></span>
+                                    </span>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-md-6 pull-right">
+                            <div class="form-group">
+                                <label class="control-label">occurred To</label>
+                                <div class="input-group" id="datetimepicker-To">
```

Comment:
The last I talked with @jbranigan, we wanted to not include occurred_to in the interface, as we're not planning on handling aoristic data in DRIVER. We do want to support it in ashlar though, so we'd only have a single datetime input and set both occurred_to and occurred_from to be equal. I say we either do this, or, if we decide we do want to support it, add a checkbox for "Occurred to is the same as Occurred from" and have it checked by default, since the vast majority of records will not have different from/to datetimes.